hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImplTest.java

```diff
@@ -191,7 +193,8 @@ void nullConstructorArgsTest() {
     @DisplayName("Null pre-handle args throw NPE")
     @SuppressWarnings("DataFlowIssue") // Suppress the warning about null args
     void nullPreHandleArgsTest() {
-        final List<Transaction> list = List.of(new SwirldTransaction(new byte[10]));
+        final List<Transaction> list = List.of(
+                new PayloadWrapper(new OneOf<>(PayloadOneOfType.APPLICATION_PAYLOAD, Bytes.wrap(new byte[10]))));
```

Comment:
Yikes! Is it possible to have a nice factory method?

Reply:
Done